streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/ApplicationServiceImpl.java

```diff
@@ -519,6 +529,21 @@ public boolean existsByTeamId(Long teamId) {
         return baseMapper.existsByTeamId(teamId);
     }
 
+    @Override
+    public boolean existsRunningJobByClusterId(Long clusterId) {
+        boolean exists = baseMapper.existsRunningJobByClusterId(clusterId);
+        if (!exists) {
+            for (Application application:FlinkTrackingTask.getAllTrackingApp().values()) {
```

Comment:
no spaces around the colon

Reply:
> no spaces around the colon

Thanks for your review, I will improve it later